frontends/election-manager/src/app.test.tsx

```diff
@@ -302,7 +302,61 @@ test('authentication works', async () => {
   );
 });
 
-test('printing ballots, print report, and test decks', async () => {
+test('L&A (logic and accuracy) flow', async () => {
+  // TODO: Move common setup logic to a beforeEach block
+  jest.useFakeTimers();
+  const card = new MemoryCard();
+  const hardware = MemoryHardware.buildStandard();
+  const printer = fakePrinter();
+  const storage = await createMemoryStorageWith({
+    electionDefinition: eitherNeitherElectionDefinition,
+  });
+  const { container, findByText, getAllByText, getByText } = render(
+    <App card={card} hardware={hardware} printer={printer} storage={storage} />
+  );
```

Comment:
Love the new test!

A few other best practices that I didn't mention before since you were modifying existing tests:
- Use `screen.getBy...` to access the query functions instead of using the result of `render` (https://testing-library.com/docs/queries/about/#screen)
- Use the userEvent library instead of fireEvent (https://testing-library.com/docs/user-event/intro/#difference-to-fireevent)

Reply:
Nice, thank you for these! I really like these practices, testing from the perspective of the user / screen.

Side note: We're not on the latest version of React Testing Library, so I couldn't use the latest syntax:

```
const user = userEvent.setup();
await user.click(...);
```

And instead had to go with:

```
await userEvent.click(...);
```

Reply:
Woah I didn't even know about the latest syntax!